src/frontend/shortlist/src/components/layout/signup_login/__tests__/LoginSignup.test.js

```diff
@@ -0,0 +1,317 @@
+/*
+TODO(All these vue files do not have props that correspond to the API, needs to be added)
+The following tests conclude all major component testing.
+*/
+
+import { describe, it, expect, vi } from "vitest";
+import { mount } from "@vue/test-utils";
+import Signup from "../Signup.vue";
+import Login from "../Login.vue";
+import Logo from "../Logo.vue";
+import LoggedOut from "../LoggedOut.vue";
+import ModalFullScreen from "./../../ModalFullScreen.vue";
+// import axios from "axios";
+import { createTestingPinia } from "@pinia/testing";
+
+describe("Login.vue", () => {
+  // Check if login exists
+  it("Renders with null prop", () => {
+    const componentWrapper = mount(Login, {
+      props: {
+        /*
+        TODO(Add API variables here)
```

Comment:
login.vue does not have prop and since for unit test we do not consider axio call, I don't think we need this whole block: props: ...

Reply:
It does not have. But I believe it will have to have some to match API calls? I think Daniel did not know the frontend codes do not have props for login. I left them there as space for him to add API tests later.

Reply:
I think he mentioned we are not doing API call check in unit test?

Reply:
Nope, but you in the meeting on Friday mentioned that we somehow need to write something to mimic API testing, and then he clarified this morning that he didn't need me to do any API writing, rather just needed me to leave space and do as much component testing as possible. That's why I left the comments there so people know where to pick things up from there